metadata-db-app/src/cmr/metadata_db/migrations/070_modify_subscriptions_table.clj

```diff
@@ -0,0 +1,103 @@
+(ns cmr.metadata-db.migrations.070-modify-subscriptions-table
+  (:require [clojure.java.jdbc :as j]
```

Comment:
nit: start the namespaces on a separate line.

Reply:
Done.